test/controllers/ApplicationControllerSpec.scala

```diff
@@ -35,16 +35,16 @@ import scala.concurrent.Future
 class ApplicationControllerSpec extends PlaySpec with GuiceOneServerPerSuite with MockitoSugar with BeforeAndAfterEach with AtedTestHelper {
 
   implicit val hc: HeaderCarrier = HeaderCarrier()
-  val injview = app.injector.instanceOf[views.html.unauthorised]
-  val injview2 = app.injector.instanceOf[views.html.unauthorisedAssistantOrg]
-  val injview3 = app.injector.instanceOf[views.html.unauthorisedAssistantAgent]
+  val injectedViewInstance = app.injector.instanceOf[views.html.unauthorised]
```

Comment:
dont use numbers. Its bad practice again. So you could call this one unauthorisedInjectedViewInstance or something along those lines. Dont use numbers if you can help it.